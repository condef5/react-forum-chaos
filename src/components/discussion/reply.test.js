import React from "react";
import { render } from "@testing-library/react";

import Reply from "./reply";

test("Reply component", () => {
  const fakeFunction = jest.fn();
  const { asFragment } = render(
    <Reply
      user={{ id: 1, username: "" }}
      discussionId={1}
      handleNewComment={fakeFunction}
      parentComment={1}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
