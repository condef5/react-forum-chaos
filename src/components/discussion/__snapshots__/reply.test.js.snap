// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reply component 1`] = `
<DocumentFragment>
  .emotion-0 {
  background: none;
  border: 1px solid #eaeaea;
  border-radius: .25rem;
  box-sizing: border-box;
  display: block;
  font-size: 1rem;
  padding: .5rem;
  width: 100%;
  height: 100px;
}

.emotion-0:focus {
  outline: none;
  border-color: #444444;
}

.emotion-1 {
  background-color: #f60;
  border: 1px solid #f60;
  border-radius: .25rem;
  color: white;
  cursor: pointer;
  font-size: .8rem;
  padding: .75rem 0;
  -webkit-transition: all 200ms ease;
  transition: all 200ms ease;
  text-align: center;
  text-transform: uppercase;
  width: 10rem;
}

.emotion-1:hover {
  background-color: #fff;
  color: #f60;
}

<form>
    <textarea
      aria-label="Reply"
      class="emotion-0"
      placeholder="Write Reply"
      type="text"
    />
    <button
      class="emotion-1"
      type="submit"
    >
      Reply
    </button>
  </form>
</DocumentFragment>
`;
